core/src/main/c/share/jit/impl/x86.h

```diff
@@ -503,6 +503,28 @@ namespace questdb::x86 {
         return r.as<Gpq>();
     }
 
+    inline Gpq int128_eq(Compiler &c, const Xmm &lhs, const Xmm &rhs) {
```

Comment:
`vpcmpeqb` - this is an avx instruction.
for sse2 could be something like:
```
eq_sse2(long long __vector(2), long long __vector(2)):                      # @eq_sse2(long long __vector(2), long long __vector(2))
        pcmpeqb xmm0, xmm1
        pmovmskb        eax, xmm0
        cmp     eax, 65535
        sete    al
        ret
```

for sse4.1 (can be used for avx2 code generation):
```
eq_sse4.1(long long __vector(2), long long __vector(2)):
        vpxor   xmm0, xmm1, xmm0
        vptest  xmm0, xmm0
        sete    al
        ret
```

Reply:
oops, thanks for catching this. Earlier on I didn't realize the `v` prefixed instructions were AVX so I was using them interchangeably with the corresponding SSE2 instructions; once I realized what was going on, I fixed most of the references in this file on my own, but it seems I missed this one